edx-solutions/xblock-drag-and-drop-v2
# Taniwha/bb 2681 fix ytp x blocks translations

This PR brings in the Arabic translates from the gingko-rg and master branches of the raccoongang repository, along with a RTL fix. It also backports some translations from master.

**JIRA tickets**: https://tasks.opencraft.com/browse/BB-2681

**Discussions**: N/A

**Dependencies**: None. However, this PR is taken from the v2.1.8 tag and thus will need a new branch to support ironwood.

**Screenshots**: N/A

**Sandbox URL**: N/A

**Merge deadline**: None

**Testing instructions**:

1. install as per usual.
2. in studio add compoenent* problem -> advanced -> drag and drop. It may be a good idea to mare all instructor-input text to for clarity as to what is and is not translated.
3. publish
4. in lms, go to unit, ensure drag and drop is working
5. switch to Arabic (ar)]
6. reload drag and drop problem page, check translations: in particular, submit button, feedback messages, keyboard help, attempts remaining

**Author notes and concerns**:

As noted above, these translations are needed in an ironwood installation thus the PR is branched from the v2.1.8 tag. 

**Reviewers**
